src/js/components/ThumbsRating/__tests__/ThumbsRating.tsx

```diff
@@ -38,6 +38,15 @@ describe('ThumbsRating', () => {
     expect(container).toMatchSnapshot();
   });
 
+  test('StarRating renders outside grommet wrapper', () => {
+    const { container, getByTestId } = render(
+      <ThumbsRating name="test" data-testid="ThumbsRating-icon" />,
+    );
+
+    expect(getByTestId('ThumbsRating-icon')).toBeTruthy();
```

Comment:
```suggestion
    const { container } = render(
      <ThumbsRating name="test" />,
    );

```
Can we simplify this test?